frameworks/person-escort-record/README.md

```diff
@@ -97,6 +105,17 @@ Question files should live in [`questions/`](./questions) and be stored in a fla
 - `validations` - list of [validations](#question-validation) that need to be applied to this question
   - `type` **(required)** - the validation error key
   - `message` - custom text that will be displayed in the form
+  - `value` - further data required for some [validation types](#question-validation).
```

Comment:
```suggestion
  - `arguments` - array of validator arguments required for some [validation types](#question-validation), eg. `{ type: 'minlength', arguments: [24] }`
```

Totally stealing the idea from the [HO Form wizard](https://github.com/UKHomeOffice/passports-form-wizard#field-options)...